examples/models/llava/runner/llava_runner.h

```diff
@@ -37,41 +37,12 @@ class LlavaRunner : public MultimodalRunner {
       std::function<void(const std::string&)> token_callback = {},
       std::function<void(const ::executorch::extension::llm::Stats&)>
           stats_callback = {});
-
-  /**
-   * Prefill an LLaVA Module with the given images input.
-   * @param images The image input to LLaVA.
-   * @param start_pos The starting position in KV cache of the input in the LLM.
-   * It's passed as reference and will be updated inside this function.
-   * @return The error status of prefilling images.
-   */
   Error prefill_images(std::vector<Image>& images, int64_t& start_pos);
```

Comment:
You can delete these apis since they are in the base class

Reply:
Probably everything https://github.com/pytorch/executorch/blob/f9764dc6f2d33c018fc6842980f6be521a8e8ed0/examples/models/llava/runner/llava_runner.h#L31-L52